pkg/utils/ddp.go

```diff
@@ -61,6 +81,9 @@ func GetDDPProfiles(dev string) (string, error) {
 	return getDDPNameFromStdout(stdout.Bytes())
 }
 
+// ErrProfileNameNotFound error when DDPTool is supported, but package name is empty
+var ErrProfileNameNotFound = errors.New("DDP profile name not found")
```

Comment:
https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin/pull/387/files#diff-83c02f96251b0c36e761108850cbbbeb1eeae8a9d6fb714b473ea8ac1242631aR59

Reply:
error message is used here to check if error matches

Reply:
I don't understand sorry, we just set a var for the string to check at L59. 
instead of just hardcoding it. 

no one returns the error, since it's a standard error that could occur, we use it for comparison only as a valid error. 
since method only returns true or false.

Reply:
errors.Is() check will not return true in this case

see[1].

we should explicitly return `ErrProfileNameNotFound` in GetDDPProfiles or wrap it in some other error

[1] https://go.dev/play/p/4sv-xe4SPd5

Reply:
thank you for the extra clarification
after looking at the go playground code, I see your point. 
it would never return correctly when doing error.Is()

We could just wrap the error then do the comparison/check, so error.Is() will respond with correct response. 
or 
we could update function to return bool and error msg from this func ``IsDDPToolSupportedByDevice``

do you have a preference? I think we could just update the comparison and continue as done before when I had it as ``err != ErrProfileNameNotFound``

Reply:
what i would do is:

in GetDDPProfile , if err happend , check if it contains the string "DDP profile name not found" then either wrap it with `ErrProfileNameNotFound` or return `ErrProfieNameNotFound`

Reply:
I've updated so that ``ErrProfileNameNotFound`` is returned